src/main/java/app/chatbot/Duke.java

```diff
@@ -0,0 +1,92 @@
+package app.chatbot;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import app.command.Command;
+import app.command.LoadCommand;
+import app.command.SaveCommand;
+import app.task.TaskList;
+
+public class Duke {
```

Comment:
> Well done on the GUI, it looks amazing! The different packages and class show good OOP paradigms and effort on your iP.
> 
> There are only minor JavaDoc and style issues. Mainly `white spaces` and `unused variables`. The missing JavaDocs for the methods can all be one-liners so they are not egregious at all. I have highlighted some of these trivial errors but seeing as how they repeat, I have attached the [Full Checkstyle report here](https://github.com/nus-cs2103-AY2223S2/ip/files/10607890/main.2.docx) instead of commenting one by one.

Much appreciated for the feedback! Concur on the issues raised as well - will address them in the next commit.
The note on the BufferReader was helpful and prompted some extra learning at my end as well.